src/components/Footer.js

```diff
@@ -3,20 +3,22 @@ import styled from "styled-components";
 import Calendar from "../icons/Calendar";
 import Faq from "../icons/Faq";
 import Medicine from "../icons/Medicine";
+import HomeButton from "../icons/HomeButton";
 
 const FooterPage = styled.div`
   display: flex;
   width: 375px;
```

Comment:
I assume the footer is supposed to take up all the available width space, why not just use width: 100vw/100% in this case ? It should do the same :)

Reply:
True, thank you :)